docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
+        "maxBytesInMemory": 0,
+        "skipBytesInMemoryOverheadCheck": false,
+        "maxRowsPerSegment": 5000000,
+        "maxTotalRows": null,
+        "intermediatePersistPeriod": "PT10M",
+        "maxPendingPersists": 0,
+        "indexSpec": {
+            "bitmap": {
+                "type": "roaring"
+            },
+            "dimensionCompression": "lz4",
+            "stringDictionaryEncoding": {
+                "type": "utf8"
+            },
+            "metricCompression": "lz4",
+            "longEncoding": "longs"
+        },
+        "indexSpecForIntermediatePersists": {
+            "bitmap": {
+                "type": "roaring"
+            },
+            "dimensionCompression": "lz4",
+            "stringDictionaryEncoding": {
+                "type": "utf8"
+            },
+            "metricCompression": "lz4",
+            "longEncoding": "longs"
+        },
+        "reportParseExceptions": false,
+        "handoffConditionTimeout": 0,
+        "resetOffsetAutomatically": false,
+        "segmentWriteOutMediumFactory": null,
+        "workerThreads": null,
+        "chatThreads": null,
+        "chatRetries": 8,
+        "httpTimeout": "PT10S",
+        "shutdownTimeout": "PT80S",
+        "offsetFetchPeriod": "PT30S",
+        "intermediateHandoffPeriod": "P2147483647D",
+        "logParseExceptions": false,
+        "maxParseExceptions": 2147483647,
+        "maxSavedParseExceptions": 0,
+        "skipSequenceNumberAvailabilityCheck": false,
+        "repartitionTransitionDuration": "PT120S"
+    },
+    "ioConfig": {
+        "topic": "social_media",
+        "inputFormat": {
+            "type": "json",
+            "keepNullColumns": false,
+            "assumeNewlineDelimited": false,
+            "useJsonNodeReader": false
+        },
+        "replicas": 1,
+        "taskCount": 1,
+        "taskDuration": "PT3600S",
+        "consumerProperties": {
+            "bootstrap.servers": "localhost:9094"
+        },
+        "autoScalerConfig": null,
+        "pollTimeout": 100,
+        "startDelay": "PT5S",
+        "period": "PT30S",
+        "useEarliestOffset": true,
+        "completionTimeout": "PT1800S",
+        "lateMessageRejectionPeriod": null,
+        "earlyMessageRejectionPeriod": null,
+        "lateMessageRejectionStartDateTime": null,
+        "configOverrides": null,
+        "idleConfig": null,
+        "stream": "social_media",
+        "useEarliestSequenceNumber": true
+    },
+    "context": null,
+    "suspended": false
+}
+  ```
+</details>
+
+### Get Supervisor status
+
+#### URL
+<code className="getAPI">GET</code> `/druid/indexer/v1/supervisor/{supervisorId}/status`
+
+Retrieves a current status report of the Supervisor with the provided ID. The report contains the state of the tasks managed by the Supervisor and an array of recently thrown exceptions. The possible `state` values are: [`PENDING`, `RUNNING`, `SUSPENDED`, `STOPPING`, `UNHEALTHY_SUPERVISOR`, `UNHEALTHY_TASKS`].
```

Comment:
```suggestion
Retrieves the current status report for a single supervisor. The report contains the state of the supervisor tasks and an array of recently thrown exceptions.
```
You don't need to mention the possible state values here since you already provide that information in the properties table.